resources/dicts/relationship_events/normal_interactions/admiration/decrease.json

```
@@ -1,152 +1,215 @@
-			"elder",
-			"apprentice",
-			"warrior",
-			"mediator apprentice",
-			"mediator",
-			"medicine cat apprentice",
-			"medicine cat",
-			"deputy",
-			"leader"
-		]
-	},
-	{
-		"id": "admire_adult_de_med1",
-		"interactions": [
-			"m_c saw r_c miss an easy catch today.",
-			"m_c is disappointed in r_c's subpar work lately.",
-			"m_c is let down by r_c's sloppy performance during a sparring session.",
-			"r_c was supposed to be showing m_c a cool battle move but messed up so badly it made m_c cringe."
-		],
-		"main_status_constraint": [
-			"apprentice",
-			"warrior",
-			"deputy",
-			"leader"
-		],
-		"random_status_constraint": [
-			"apprentice",
-			"warrior",
-			"deputy",
-			"leader"
-		]
-	},
-	{
-		"id": "admire_adult_de_med2",
-		"interactions": [
-			"m_c has noticed r_c isn't doing a lot of work lately.",
-			"m_c is disappointed in r_c's lack of initiative.",
-			"m_c notices that r_c seems to disappear whenever difficult tasks are being discussed.",
-			"m_c has noticed r_c's been a lot lazier lately."
-		],
-		"main_status_constraint": [
-			"elder",
-			"apprentice",
-			"warrior",
-			"deputy",
-			"leader",
-			"medicine cat",
-			"mediator"
-		],
-		"random_status_constraint": [
-			"apprentice",
-			"warrior",
-			"mediator apprentice",
-			"mediator",
-			"medicine cat apprentice",
-			"medicine cat",
-			"deputy",
-			"leader"
-		]
-	},
-	{
-		"id": "admire_de_high1",
-		"intensity": "high",
-		"interactions": [
-			"m_c heard an ugly rumor about r_c.",
-			"m_c overheard r_c spreading a hurtful rumor."
-		]
-	},
-	{
-		"id": "admire_de_high2",
-		"intensity": "high",
-		"interactions": [
-			"m_c had a difficult conversation with r_c, and they both ended up resentful of each other.",
-			"m_c was noticeably distracted while r_c was trying to tell a story.",
-			"m_c and r_c have a long discussion about goals and aspirations, and find that they don't agree on many things.",
-			"m_c spent a long time talking with r_c and was disappointed by r_c's lack of interest.",
-			"m_c asks r_c about something complicated and regrets it almost immediately when r_c misses the point."
-		],
-		"reaction_random_cat": {
-			"admiration": "decrease"
-		}
-	}
-]
+[
+	{
+		"id": "admire_de_low1",
+		"intensity": "low",
+		"interactions": [
+			"m_c saw r_c being rude.",
+			"m_c heard r_c say something mean.",
+			"m_c is annoyed by r_c's negative comments.",
+			"m_c notices r_c failed to complete a simple task.",
+			"m_c hopes {PRONOUN/m_c/subject} just misheard r_c's rude remark.",
+			"r_c keeps asking m_c silly questions.",
+			"m_c wonders why r_c is so pouty today.",
+			"r_c sneezed on m_c.",
+			"m_c was woken up early by r_c.",
+			"m_c notices that r_c overslept.",
+			"r_c tells m_c to smile more.",
+			"m_c had a funny dream about r_c and can't take {PRONOUN/r_c/object} seriously now.",
+			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on {PRONOUN/r_c/poss} pelt."

```

```suggestion
			"r_c is talking to m_c, but {PRONOUN/m_c/subject} can't stop noticing the mud on r_c's pelt."

```
In this instance, if both cats had the same pronouns then it could be misinterpreted as m_c having mud on their pelt, rather than r_c.